docs/rbd/deploy.md

```diff
@@ -524,3 +524,36 @@ module is enabled on the nodes running ceph-csi attachers.
 
 If custom image is built for the rbd-plugin instance, make sure that it contains
 `cryptsetup` tool installed to be able to use encryption.
+
+## Changed Block Tracking (CBT)
+
+> **Warning**: Requires Ceph version that supports RBD snap diff by ID feature.
+For details, see ceph tracker:
+["diff-iterate by snap ID"](https://tracker.ceph.com/issues/65720).
+
+Ceph-CSI implements Changed Block Tracking (CBT) for RBD volumes using
+the SnapshotMetadataService (SMS) APIs. This feature enables efficient
+and reliable differential backup of data stored in CSI volumes.
+
+The feature is exposed through the CSI Controller Server's
+SnapshotMetadataService APIs with two primary operations:
+
+* `GetMetadataAllocated`:
+
+   * Streams metadata about allocated blocks in a snapshot
+   * Useful for full backup operations
+   * Returns block ranges that contain actual data
+
+* `GetMetadataDelta`:
+
+   * Streams metadata about block differences between two snapshots
+   * Optimal for incremental backups
+   * Returns only blocks that changed between snapshots
```

Comment:
This doc is for someone who wants to deploy and test it out, can we add relevant steps for the same?

Reply:
> This doc is for someone who wants to deploy and test it out, can we add relevant steps for the same?

Since ceph-csi-operator is the one deploying the sidecar, I'll add a link here to that doc when its ready over there.